.storybook/main.js

```diff
@@ -10,9 +10,10 @@ module.exports = {
     builder: 'webpack5',
   },
   stories: [
-    '../src/app/legacy/components/**/*.stories.jsx',
-    '../src/app/legacy/containers/**/*.stories.jsx',
-    '../src/app/pages/**/*.stories.jsx',
+    '../src/app/legacy/components/**/*.stories.(t|j)sx',
+    '../src/app/legacy/containers/**/*.stories.(t|j)sx',
+    '../src/app/components/**/*.stories.(t|j)sx',
+    '../src/app/pages/**/*.stories.(t|j)sx',
```

Comment:
Forgot to fix those warnings - https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#correct-globs-in-mainjs
```suggestion
    '../src/app/legacy/components/**/*.stories.@(t|j)sx',
    '../src/app/legacy/containers/**/*.stories.@(t|j)sx',
    '../src/app/components/**/*.stories.@(t|j)sx',
    '../src/app/pages/**/*.stories.@(t|j)sx',
```